plugin/trino-resource-group-managers/src/main/resources/db/migration/postgresql/V5__add_primary_key_to_selectors.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE selectors ADD COLUMN id BIGSERIAL PRIMARY KEY;
```

Comment:
You can also use GENERATED since PostgreSQL < 10 isn't supported anymore and serials have some known issues.

```suggestion
ALTER TABLE selectors ADD COLUMN id BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY;
```

Reply:
@hashhar Thanks for your suggest! Done!

Reply:
> You can also use GENERATED since PostgreSQL < 10 isn't supported anymore and serials have some known issues.

> 
But unit test will faile when using GENERATED:
`
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at or near "GENERATED"
  Position: 56
`

Reply:
You need to change the version being used for tests at https://github.com/trinodb/trino/blob/d72b25ece9c721291d0d4d5b0f32be82400d8990/plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsPostgresqlFlywayMigration.java#L27

9.x isn't supported anymore and we've already moved other tests to use 10.

Reply:
I see. thx. I use pg 10.0 to test.